modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,50 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which can be configured by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
+
+.Prerequisites
+
+* You must have the {ServerlessOperatorName} and Knative Serving installed on your {product-title} cluster.
+
+.Procedure
+
+. Create a Knative service that contains the label or annotation that you want to propagate to the {product-title} route:
++
+.Example service YAML
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+  labels:
+    <label_name>: <label_value>
+  annotations:
+    <annotation_name>: <annotation_value>
+...
+----
++
+.Example kn command
```

Comment:
Would it make sense to break this up into two steps or move the service YAML into the prereqs? That way, this "create" step is only associated with the command.

Reply:
They are both ways to create though, that's kind of the point 🙂 you can either create a YAML object or use the kn command. Updating the .headings to make this clearer

Reply:
Eventually we'll have a better strategy around documenting CLI vs YAML vs ODC but I think that's out of scope for this PR at the minute, I'd prefer to just show both options here for now.

Reply:
> They are both ways to create though, that's kind of the point 🙂 you can either create a YAML object or use the kn command. Updating the .headings to make this clearer

In that case, ISG asks for a different construction:

> **Handling nonsequential actions**
> When a procedure has mutually exclusive alternatives or any kind of branching condition, do not present the alternatives as sequential steps. Describe the branching condition in a single step, using a sublist or subprocedures to present the alternatives. Do not insert the word or between alternatives.

Reply:
Added these as a sublist instead (next commit), lmk if that's better 🙂 thanks